src/main/java/club/sk1er/patcher/mixins/bugfixes/optifine/RenderGlobalMixin_FixSkyVBOs.java

```diff
@@ -0,0 +1,24 @@
+package club.sk1er.patcher.mixins.bugfixes.optifine;
+
+import net.minecraft.client.renderer.RenderGlobal;
+import org.objectweb.asm.Opcodes;
+import org.spongepowered.asm.mixin.Dynamic;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.ModifyVariable;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+@Mixin(RenderGlobal.class)
+public class RenderGlobalMixin_FixSkyVBOs {
+    @Dynamic("OptiFine")
+    @ModifyVariable(method = "renderSky(Lnet/minecraft/client/renderer/WorldRenderer;FZ)V", at = @At("STORE"), ordinal = 2)
+    private int patcher$distanceOverride(int k) {
+        return 384;
+    }
+
+    @Dynamic("OptiFine")
+    @Redirect(method = "renderSky(FI)V", at = @At(value = "FIELD", target = "net/minecraft/client/renderer/RenderGlobal.vboEnabled:Z", opcode = Opcodes.GETFIELD, ordinal = 3))
```

Comment:
Could you potentially look into replacing the `ordinal = 3` with a `@Slice`? E.g. starting at the reference to `renderDistanceChunks`, which is unique and happens before the desired `vboEnabled` access.
I'm not entirely sure if that Slice is expressive enough to be more stable than the ordinal, but it's worth considering, and perhaps you can think of a better one.

Reply:
Using `@Slice` seems at least more legible code-wise to me so I've gone ahead and done this instead. Thank you for the suggestion + review.